Assets/MixedRealityToolkit/Utilities/StandardShader/BaseMeshOutline.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// Abstract component to encapsulate common functionality around outline components. 
+    /// </summary>
+    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_MRTKStandardShader.html#mesh-outlines")]
+    public abstract class BaseMeshOutline : MonoBehaviour
+    {
+        /// <summary>
+        /// The material used to render the outline. Outline materials should normal have "Depth Write" set to Off and "Vertex Extrusion" enabled.
+        /// Most MRTK/Standard features should work as an outline material, but it is recommended to use a simple unlit material.
```

Comment:
So I'm still trying to parse this whole PR. But is the outline feature in the scripts or in the MRTK standard shader?

All of the Outline*.mat files in this PR are MRTK standard shader. Why not make them simpler as just unlit as this comment suggests?

Reply:
Ah, this comment is misleading. Thanks for pointing that out, I'll fix this! It should read that you should use a MRTK/Standard material but it's recommend to make the MRTK/Standard material as bare bones as possible (in other words unlit).